### DISCUSSION TITLE
Simple Suggestion

### DISCUSSION THREAD

**Discussion by ruribev** [nijnstein/BLAST-Documentation#4] [Ideas]
Hello!

I notice that may be easy and helpful adding "Assambly Definition Assets" to BLAST, here are some good videos on the topic: 
https://www.youtube.com/watch?v=eovjb5xn8y0 
https://www.youtube.com/watch?v=HYqOSkHI674

I am adding them manually but for future updates it will be much easier if it is already part of the package. From other developers I understand it can be a bit tricky for some reason but having those make new code to compile way faster!

Hope this helps make BLAST better.

All the best,
Rodrigo

**Comment by nijnstein** [comment]
ill be adding this in the next release, thanks for the suggestion
